django/http/request.py

```diff
@@ -230,6 +288,17 @@ def body(self):
         if not hasattr(self, '_body'):
             if self._read_started:
                 raise RawPostDataException("You cannot access body after reading from request's data stream")
+
```

Comment:
Actually, it seems to me that this check is incorrect. Here is my original proposal:

> FORM_DATA_MAX_FIELDS_TOTAL_LENGTH -- the maximum total length of the posted fields (both for 
> "application/x-www-form-urlencoded" and "multipart/form-data"). This is the sum of the lengths of all field-names 
> and field-values). Reading from the input socket should not continue after this length-limit has been exceeded. 
> (no limit by default)

It seems to me that this check should not be done in case of "multipart/form-data" mime type. Am I mistaken?